lightning/src/util/config.rs

```diff
@@ -305,6 +305,9 @@ pub struct UserConfig {
 	/// If this is set to false, we do not accept inbound requests to open a new channel.
 	/// Default value: true.
 	pub accept_inbound_channels: bool,
+	/// If this is set to true, the user needs to manually accept inbound requests to open a new channel.
+	/// Default value: false.
```

Comment:
nit: Would be good to at least link to `Event::OpenChannelRequest` here.

Reply:
Sure, I'll add that.